vietdoan/broadway/css/custom.css

```diff
@@ -0,0 +1,121 @@
+html{
+  font-family: 'Raleway', serif;
+}
+body {
+  margin: 0;
+}
+
+h1 {
+  margin: 0;
+}
+.bg-black {
+  background-color: #333333;
+  color: white;
+  padding: 20px 0px;
+  font-size: 12px;
+  font-weight: bolder;
+}
+.bg-black ul, footer{
+  margin: 0 0 0 200px;
+  padding: 0px;
+}
+.bg-black ul li {
+  text-transform: uppercase;
+  display: inline;
+  margin-right: 25px;
+}
+
+.homepage {
+  background-image: url('../images/bg.jpg');
+  background-size: cover;
+  background-position: center center;
+  text-align: center;
+  height: 600px;
+  padding-top: 180px;
+}
+
+.homepage h1 {
+  color:  #333333;
+  font-size: 50px;
+  font-weight: bold;
+  text-transform: uppercase;
+  margin-bottom: 90px;
+}
+
+.btn-black  {
+  background-color: #333333;
+  color: white;
+  padding: 15px 40px;
+  text-decoration: none;
+  font-size: 18px;
+  font-weight: bolder;
+}
+
+.btn-white {
+    border: 1px solid black;
+    font-size: 10px;
+    text-decoration: none;
+    padding: 10px 40px;
+    color: black;
+    font-weight: 200;
+    text-transform: uppercase;
+}
+
+.content {
+  text-align: center;
+  margin-bottom: 100px;
+}
+.row {
+  margin: 0 auto;
+  width: 81%;
+  display: table;
+}
+.col {
```

Comment:
Để `inline-block` nó sẽ wrap content, nghĩa là thử để phần text dài ra nó sẽ rất xấu. Với lại code responsive thì cách tốt nhất và tiện nhất a thấy là dùng [flex-box](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)